core/Pos/Binary/Core/Types.hs

```diff
@@ -61,3 +61,32 @@ instance Bi T.SharedSeed where
 instance Bi T.ChainDifficulty where
     get = label "ChainDifficulty" $ T.ChainDifficulty . getUnsignedVarInt <$> get
     put (T.ChainDifficulty c) = put (UnsignedVarInt c)
+
+instance Bi T.BlockVersionData where
+    get = label "BlockVersionData" $ do
```

Comment:
@gromakovsky why don't we use default instances for serialization? are there important causes for such decision?